packages/hooks/src/useCFDDemoAccounts.ts

```diff
@@ -0,0 +1,9 @@
+import useCFDAccounts from './useCFDAccounts';
+
+const useCFDDemoAccounts = () => {
+    const cfd_accounts = useCFDAccounts();
+
+    return cfd_accounts.filter(account => account.account_type === 'demo');
```

Comment:
```suggestion
    const cfd_demo_accounts = cfd_accounts.filter(account => account.account_type === 'demo');

    return cfd_demo_accounts;
```
@shayan-deriv Just a suggestion, I personally prefer creating helper variables with meaningful names to make the code more readable instead of directly retuning the result of some operation.